test/controllers/AdminControllerSpec.scala

```diff
@@ -259,7 +259,30 @@ class AdminControllerSpec
       }
     }
 
-    "must must state to rejected when state is Submitted and Approved request is sent" in {
+    "must return not found when state is not Submitted or Rejected, and update request is sent" in {
```

Comment:
"must return not found when state is not Submitted, and update request is sent"

Reply:
fixed